addons/l10n_in/models/account.py

```diff
@@ -31,6 +31,11 @@ def name_get(self):
 class AccountMoveLine(models.Model):
     _inherit = "account.move.line"
 
+    def init(self):
+        self._cr.execute("""SELECT indexname FROM pg_indexes WHERE indexname='account_move_line_move_product_idx'""")
+        if not self._cr.fetchone():
+            self._cr.execute("""CREATE INDEX account_move_line_move_product_idx ON account_move_line (move_id, product_id)""")
```

Comment:
There is a function in `odoo.tools` for that, simply add a `from odoo import tools` and
```suggestion
        tools.create_index(self._cr, 'account_move_line_move_product_idx',
                           self._table, ['move_id', 'product_id])
```